sentry-core/src/test/java/io/sentry/core/SentryClientTest.kt

```diff
@@ -403,19 +403,6 @@ class SentryClientTest {
         assertEquals(transportGate, sentryOptions.transportGate)
     }
 
-    @Test
-    fun `when transport gate is null, it should init an always on transport gate`() {
-        val sentryOptions: SentryOptions = SentryOptions().apply {
-            dsn = dsnString
-        }
-
-        val connection = mock<AsyncConnection>()
-        SentryClient(sentryOptions, connection)
-
-        assertNotNull(sentryOptions.transportGate)
-        assertTrue(sentryOptions.transportGate!!.isConnected)
-    }
-
```

Comment:
`m`: Why did we remove this test when we fix linting?

Reply:
transport gate is never null, its NoOp, and there's a test for this already.
it was warning that `!!` is not necessary, but instead of fixing it, I removed the test which is useless after transportGate becoming NoOp in a different PR